src/layouts/InterfaceLayout/containers/SendOfflineContainer/SendOfflineContainer.vue

```diff
@@ -326,6 +327,26 @@ export default {
       this.createDataHex(null, null, newVal);
     }
   },
+  mounted() {
+    if (Object.keys(this.linkQuery).length > 0) {
+      const { data, to, value, gaslimit, gas, tokensymbol } = this.linkQuery;
```

Comment:
did you get a hance to check what will happen if some of the values are undefined, I think we should check whether they are undefined and set to default values

Reply:
Yeah. I tried it with just a data field and it changed just that iirc. But I'll try it again

Reply:
@kvhnuke new push checks values now.